py4j-python/src/py4j/tests/java_dir_test.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import unicode_literals, absolute_import
+
+
+from py4j.java_gateway import JavaGateway, GatewayParameters, java_import,\
+    UserHelpAutoCompletion
+from py4j.tests.java_gateway_test import (
+    start_example_app_process)
+from contextlib import contextmanager
+from nose.tools import eq_
```

Comment:
OK, I'll update for that. Actually I have started using py.test for most stuff and its assert rewriting is really nice, can't believe I wasn't using it earlier. So I'll udpate the code to use assert directly (can't use self.assertEquals because this code is not within a testcase class)